pkg/distro/image_config.go

```diff
@@ -20,7 +20,8 @@ type ImageConfig struct {
 	DisabledServices    []string
 	DefaultTarget       *string
 	Sysconfig           []*osbuild.SysconfigStageOptions
-
+	// whether to create sudoer file for non root users
+	CreateUsersSodoerFile *bool
```

Comment:
The option name has a typo, "sudoers" not "sodoers".
More generally though, I think it should be a bit more descriptive about what it does.  Yes, we're adding a drop-in for users in the `/etc/sudoers.d`, but functionally the purpose of the option is to enable NOPASSWD sudo-ing.

I'd rename it to something like `NopasswdSudo`.